{"cert_user_id": "brunoais@github", "next_topic_id": 2, "topic": [{"topic_id": 1604065398, "title": "Cannot connect to tor", "body": "It won't connect to the Tor network.\nThis shows:\n\n> Status: Error ([Errno 10061] No connection could be made because the target machine actively refused it.)\n\nI don't understand what is causing it does it have the wrong URL to reach the tor network?\n\nI'm using windows 10.\n", "parent_topic_uri": "1604062980_users_1Cy3ntkN2GN9MH6EaW6eHpi4YoRS2nK5Di", "added": 1470765096, "modified": 1515050605, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/526", "source_type": "github"}], "next_comment_id": 6, "comment": {"1604065398_mirrored_brunoais_github": [{"comment_id": 1, "body": "It looks like this:\n\n```\n[2016- 00:00:03,572] INFO     - Creating FileServer....\n[2016- 00:00:03,573] DEBUG    TorManager Connecting to 127.0.0.1:9051\n[2016- 00:00:04,578] ERROR    TorManager Tor controller connect error: error: [Errno 10061] No connection could be made because the target machine actively refused it. in TorManager.py line 154 > socket.py line 342\n[2016- 00:00:04,581] DEBUG    TorManager Tor proxy port 127.0.0.1:9050 check error: No connection\n[2016- 00:00:04,581] INFO     TorManager Starting Tor client tools/tor/tor.exe...\n[2016- 00:00:05,086] DEBUG    TorManager Connecting to 127.0.0.1:49051\n[2016- 00:00:06,088] ERROR    TorManager Tor controller connect error: error: [Errno 10061] No connection could be made because the target machine actively refused it. in TorManager.py line 154 > socket.py line 342\n[2016- 00:00:07,092] DEBUG    TorManager Connecting to 127.0.0.1:49051\n[2016- 00:00:08,094] ERROR    TorManager Tor controller connect error: error: [Errno 10061] No connection could be made because the target machine actively refused it. in TorManager.py line 154 > socket.py line 342\n[2016- 00:00:09,595] DEBUG    TorManager Connecting to 127.0.0.1:49051\n[2016- 00:00:10,598] ERROR    TorManager Tor controller connect error: error: [Errno 10061] No connection could be made because the target machine actively refused it. in TorManager.py line 154 > socket.py line 342\n[2016- 00:00:12,599] DEBUG    TorManager Connecting to 127.0.0.1:49051\n[2016- 00:00:13,601] ERROR    TorManager Tor controller connect error: error: [Errno 10061] No connection could be made because the target machine actively refused it. in TorManager.py line 154 > socket.py line 342\n(repeats a lot)\n```\n\nIt doesn't seem to be trying to start tor client inside that tools directory.\n", "added": 1470768405, "modified": 1470768598, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/526#issuecomment-238702257", "source_type": "github"}, {"comment_id": 2, "body": "Hum... When I use `tools\\tor\\start.bat` I either can start tor with administrator permissions or it fails to start. When I try to start without administrator permissions:\n\n```\nAug 10 05:16:55.144 [notice] Tor v0.2.7.6 (git-7a489a6389110120) running on Windows 8 with Libevent 2.0.22-stable, OpenSSL 1.0.1t and Zlib 1.2.8.\nAug 10 05:16:55.149 [notice] Tor can't help you if you use it wrong! Learn how to be safe at https://www.torproject.org/download/download#warning\nAug 10 05:16:55.167 [notice] Read configuration file \"path\\to\\zeronet\\ZeroNet\\tools\\tor\\torrc\".\nAug 10 05:16:55.171 [warn] Path for GeoIPFile (geoip\\geoip) is relative and will resolve to path\\to\\zeronet\\ZeroNet\\tools\\tor\\geoip\\geoip. Is this what you wanted?\nAug 10 05:16:55.174 [warn] Path for GeoIPv6File (geoip\\geoip6) is relative and will resolve to path\\to\\zeronet\\ZeroNet\\tools\\tor\\geoip\\geoip6. Is this what you wanted?\nAug 10 05:16:55.176 [warn] Path for DataDirectory (data) is relative and will resolve to path\\to\\zeronet\\ZeroNet\\tools\\tor\\data. Is this what you wanted?\nAug 10 05:16:55.182 [notice] Opening Socks listener on 127.0.0.1:49050\nAug 10 05:16:55.183 [notice] Opening Control listener on 127.0.0.1:49051\nAug 10 05:16:55.000 [warn] Error replacing \"data\\state\": Invalid argument\nAug 10 05:16:55.000 [warn] Unable to write state to file \"data\\state\"; will try again later\nAug 10 05:16:55.000 [warn] Error replacing \"data\\control_auth_cookie\": Invalid argument\nAug 10 05:16:55.000 [warn] Error writing auth cookie to \"data\\\\control_auth_cookie\".\nAug 10 05:16:55.000 [warn] Error creating control cookie authentication file.\nAug 10 05:16:55.000 [err] set_options(): Bug: Acting on config options left us in a broken state. Dying. (on Tor 0.2.7.6 7a489a6389110120)\n```\n", "added": 1470800019, "modified": 1470800019, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/526#issuecomment-238777736", "source_type": "github"}, {"comment_id": 3, "body": "It creates a data directory with these files:\n1. `control_auth_cookie.tmp` (contains small binary data)\n2. `lock`\n3. `state.tmp` (contains small amount of text in 6 lines. The first 3 seem to be comments, though)\n", "added": 1470825766, "modified": 1470825861, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/526#issuecomment-238870205", "source_type": "github"}, {"comment_id": 4, "body": "@HelloZeroNet anything yet?\n", "added": 1471265341, "modified": 1471265341, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/526#issuecomment-239839885", "source_type": "github"}, {"comment_id": 5, "body": "It's weird it being a firewall issue... The issue seems to be file handling of some sort...\nI just checked now (after you answer) and tor is working if I run it with administrator permissions O_O.\n", "added": 1471268359, "modified": 1471268359, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/526#issuecomment-239854542", "source_type": "github"}]}}